src/content/blog/2025-08-22-multithread-linting.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "New in ESLint v9.34.0: Multithread Linting"
+teaser: "Make use of your CPU: multithreaded linting in ESLint v9.34.0 speeds up linting large projects."
+tags:
+  - Concurrency
+  - Multithreading
+  - Performance
+draft: true
+authors:
+  - fasttime
+categories:
+  - Storytime
+---
+
+## Introduction
+
+With ESLint v9.34.0, linting can now happen in true parallel. By spawning worker threads, ESLint can process multiple files at the same time, dramatically reducing lint times for large projects.  
+
+On machines with multiple CPU cores and fast storage, this change can make a noticeable difference — especially when you're linting hundreds or thousands of files.
+
+## CLI Multithreading Support
+
+Multithreading is enabled through the new `--concurrency` flag. You can set it in a few different ways:
+
+- **A positive integer** (e.g., `4`) tells ESLint the maximum number of threads to use. It will never spawn more threads than there are files to lint.  
+- **`auto`** lets ESLint decide the optimal number of threads based on your CPU and file count.  
+- **`off`** (the default) disables multithreading, equivalent to `--concurrency=1`.
```

Comment:
```suggestion
* **A positive integer** (e.g., `4`) tells ESLint the maximum number of threads to use. It will never spawn more threads than there are files to lint.  
* **`auto`** lets ESLint decide the optimal number of threads based on your CPU and file count.  
* **`off`** (the default) disables multithreading, equivalent to `--concurrency=1`.
```